### PR TITLE
Add email service comparison table

### PR DESCRIPTION
**Pull request by redoomed1**
Changes proposed in this PR:

- Add comparison table for the Email Services page
<del>  - As marked on GitHub, this is a draft PR as wording and formatting is not set in stone. I'm unsure of what to rearrange, add, or remove, so any feedback is welcome. Here is a question as a starting point:</del>
<del>    - **Table Columns**: Are there any criteria that should be added to or removed from the current table?</del>
- Alter wording about Proton and Tuta's free plans for [clarity](https://github.com/privacyguides/privacyguides.org/pull/2512#discussion_r1560439458)
  - https://proton.me/support/more-storage-proton-drive
    > The **default** Proton Mail free storage is **500 MB**, and you can receive an _additional 500 MB_ [1 GB total] upon completing the Proton Mail-specific initial actions [explained here](https://proton.me/support/get-started-mail).
- Update Proton Unlimited description in the "Additional Functionality" section under the Proton Mail recommendation

- [x] I have disclosed any relevant conflicts of interest in my post.
- [x] I agree to grant Privacy Guides a perpetual, worldwide, non-exclusive, transferable, royalty-free, irrevocable license with the right to sublicense such rights through multiple tiers of sublicensees, to reproduce, modify, display, perform, relicense, and distribute my contribution as part of this project.
- [x] I am the sole author of this work. 
- [x] I agree to the [Community Code of Conduct](https://www.privacyguides.org/coc).

### CODE REVIEW

**Comment by redoomed1** [comment]
Changes made before marking PR as ready for review:

- Add "Recommended Providers" second-level heading
- Add sentence referencing the detailed criteria on the page (same as the one [introducing the VPN table](https://www.privacyguides.org/en/vpn/#recommended-providers))
- Remove space between number and unit (e.g., "500MB") to match other [instances](https://www.privacyguides.org/en/photo-management/#stingle) on the site
- Update instructions for disabling Proton Mail crash reporting
- Remove pricing information for Proton Unlimited and embed link to detailed support article about the paid plan instead
- Make minor grammatical changes

**Review by dngray — APPROVED** [review]

File: docs/email.md [hunk 1]
```diff
@@ -4,17 +4,23 @@ title: "Email Services"
...
+global:
+ - [randomize-element, "table tbody"]
 ---
 <!-- markdownlint-disable MD024 -->
 Email is practically a necessity for using any online service, however we do not recommend it for person-to-person conversations. Rather than using email to contact other people, consider using an instant messaging medium that supports forward secrecy.
 
 [Recommended Instant Messengers](real-time-communication.md){ .md-button }
 
-For everything else, we recommend a variety of email providers based on sustainable business models and built-in security and privacy features.
+## Recommended Providers
```
Comment (jonaharagon) [inline docs/email.md:15]: I don't think this header is necessary is it?

**Review by jonaharagon — CHANGES_REQUESTED** [review]

File: docs/email.md (hunk 1 above)
Comment (redoomed1) [reply docs/email.md:15]: Eh, perhaps it isn't. I just added it here to keep this page uniform with the other provider pages with tables.

Comment (jonaharagon) [reply docs/email.md:15]: I see that this header is consistent with the other ones, although I think that the one on the VPN page is a mistake as well, now that I'm thinking about it.

It makes sense for DNS and search engine where we have 2-3 different types of recommendations on the page, but it should be removed from this page and VPN page where there aren't recommendation sub-categories I think.

Comment (jonaharagon) [reply docs/email.md:15]: Hm... I guess we do have two different types of recommendations here too lol, although in this case we're including both in the same table so it's not the same.

Comment (redoomed1) [reply docs/email.md:15]: > two different types of recommendations here too lol, although in this case we're including both in the same table so it's not the same.

Are they really two _different types_ of recommendations in the same sense as the search engines page (primary search + metasearch) or the DNS resolvers page (self-hosted, cloud-based, and encrypted proxies)?

In the case of the email services page, I think the only distinguishing trait for the difference is that Tuta doesn't support OpenPGP (correct me if I'm wrong).

**Review by jonaharagon — APPROVED** [review]
changed my mind, LGTM 👍

**Pull request merged**